code/modules/recycling/sortingmachinery.dm

```diff
@@ -189,13 +189,15 @@
 		update_appearance()
 
 	else if(istype(item, /obj/item/boxcutter))
-		if( (item.icon_state == "boxcutter_on"))
-			unwrap_contents(user)
+		var/obj/item/boxcutter/boxcutter_item = item
+		if(boxcutter_item.on)
+			if(!attempt_pre_unwrap_contents(user, time = 5))
```

Comment:
```suggestion
			if(!attempt_pre_unwrap_contents(user, time = 0.5 SECONDS))
```

Reply:
i dont know if i want to change this to seconds, is there even any code that uses it? im worried its gonna break

Reply:
Don't worry man, seconds is literally just defined as `* 10` its just for readability.